_config.yml

```diff
@@ -13,7 +13,7 @@
 
 #theme: "hallios"
 #remote_theme: "hallios"
-minimal_mistakes_skin: "dark" # "air", "aqua", "contrast", "dark", "dirt", "neon", "mint", "plum", "sunrise"
+minimal_mistakes_skin: "hallios1" # "air", "aqua", "contrast", "dark", "dirt", "neon", "mint", "plum", "sunrise", "hallios1", "hallios2", "hallios3"
```

Comment:
We have 3 theme options here:

1 - hallios colors
2 - the most "comfort" one considering design best practices
3 - the first one + additional styling on content containers

feel free to experiment and tweak them to your desire, in opinion the color red was a little hard to fit nicely.